text/0110-track-reform.md

```diff
@@ -21,15 +22,16 @@ Another confusion is when for `@wire` decorator reactivity works fine even when
 
 ## Proposal
 
-### "Unlearn" the `@track` decorator
+### Learn the `@track` decorator when you really need it.
 
-We don't have the concept of deprecate, instead it is more accurate to call it "unlearn" because new users don't need to learn about it, and old users can just forget about it. Components that are using that decorator will just simply continue to function, and the compiler will completely ignore (remove) that decorator.
+The runtime/compiler will figure what is the list of fields that are defined by a class, and make the template react to mutations of the field at runtime.
```

Comment:
figure -> "extract all fields"

react to mutations of the field at runtime -> observe mutations of any of those fields and update the UI accordingly.